draft-ietf-trans-rfc6962-bis.md

```diff
@@ -227,6 +227,10 @@ community. The major changes are:
 - Logs IDs: each log is now identified by an OID rather than by the hash of its
   public key.
 
+- TransItem structure: this new data structure, which encapsulates most types of
+  CT data, can be used anywhere that SignedCertificateTimestampList was used in
```

Comment:
That's not quite accurate: A TransItemList can now be used everywhere a SignedCertificateTimestampList could be used in RFC6962.